packages/core/stories/style-injection/style-injection.disabled.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from "@salt-ds/core";
+import { StoryFn } from "@storybook/react";
+import { SaltProvider } from "@salt-ds/core";
+
+export default {
+  title: "Core/Style Injection",
+  component: SaltProvider,
+};
+
+export const RootDisabled: StoryFn = () => (
+  <SaltProvider enableStyleInjection={false}>
+    <Button variant="primary">Root - style injection disabled</Button>
+    <SaltProvider>
+      <Button variant="primary">Nested - style injection enabled</Button>
```

Comment:
disabled?
```suggestion
      <Button variant="primary">Nested - style injection disabled</Button>
```

Reply:
No, it's nested inside a provider that has style injection enabled (line 13).

This test should show 2 buttons with no styles as the root style provider trumps the nested style provider

Reply:
Updated the text to make clarify the expectations